core/src/main/java/com/eternalcode/core/command/configurator/config/Command.java

```diff
@@ -0,0 +1,57 @@
+package com.eternalcode.core.command.configurator.config;
+
+import net.dzikoysk.cdn.entity.Contextual;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Contextual
+public class Command {
+
+    public String name;
+    public boolean disabled;
```

Comment:
chyba lepiej zrobić `enabled` w configu

Reply:
jak dasz enabled: true to .cancel(true) ?

Reply:
enabled: true -> cancel: false
enabled: false -> cancel: true

Reply:
serio, ktoś ma robić przeczenia do booleana żeby tylko nazwa w configu była inna?

Reply:
> serio, ktoś ma robić przeczenia do booleana żeby tylko nazwa w configu była inna?

no wiem, że `!` to problematyczna sprawa, ale w tym wypadku wystarczy zmienić nazwę zmiennej i implementacje metody. (pov: piotrula użwa metod zamiast fieldów dzięki temu zmiana może się odbyć tylko w 2 linijkach dla Command)

Reply:
Rock Corp level pro engineering

Reply:
> > serio, ktoś ma robić przeczenia do booleana żeby tylko nazwa w configu była inna?
> 
> no wiem, że `!` to problematyczna sprawa, ale w tym wypadku wystarczy zmienić nazwę zmiennej i implementacje metody. (pov: piotrula użwa metod zamiast fieldów dzięki temu zmiana może się odbyć tylko w 2 linijkach dla Command)

Rock Corp level pro engineering